lib/std/builtin.zig

```diff
@@ -813,6 +813,9 @@ pub const FloatMode = enum {
 pub const Endian = enum {
     big,
     little,
+
+    pub const native = builtin.target.cpu.arch.endian();
+    pub const foreign: Endian = @enumFromInt(1 - @intFromEnum(native));
```

Comment:
I'm pretty sure this is only valid as long as mixed endian is never added. Also, translating from an enum to an int, just to go back to an enum feels kinda weird. Why not `if (native == .big) .little else .big`

Reply:
> Why not `if (native == .big) .little else .big`

Better yet: put that in a function `Endian.flip` (or `Endian.invert` or something idk) and call it here

Reply:
when would that be necessary outside right here?